scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -197,12 +200,13 @@
     true)
 
   (pending-offers [this pool-name]
-    (if (or (= pool-name (config/default-pool)) ; TODO(pschorf): Support pools
-            (= pool-name "no-pool"))
-      (let [nodes @current-nodes-atom
-            pods @all-pods-atom]
-        (generate-offers this nodes pods (controller/starting-namespaced-pod-name->pod this)))
-      []))
+    (let [nodes @current-nodes-atom
+          pods @all-pods-atom
+          starting-instances (controller/starting-namespaced-pod-name->pod this)
+          offers-all-pools (generate-offers this nodes pods starting-instances)]
+      (doseq [[pool offers] offers-all-pools]
```

Comment:
Just log offers for `pool-name`

Reply:
Bad idea. :) 
If you do that, you get the massive node object in the logged information. That's why I filter out just the size information here and log 1/100th the amount of stuff.

Reply:
I meant you probably don't want to log offers for all pools in each invocation of `pending-offers`. Just log the offers for the pool you're actively looking at. The filtering is fine.

Reply:
Aah. Got it. Ok. Also moved the TODO with this refactor.